core/src/main/java/org/jivesoftware/sparkimpl/certificates/CertificateController.java

```diff
@@ -58,6 +63,8 @@ public CertificateController(LocalPreferences localPreferences) {
 				case 2:
 					return String.class;
 				case 3:
+					return String.class;
+				case 4:
 					return Boolean.class;
 				default:
 					return String.class;
```

Comment:
You have exactly 5 columns now, correct? The `default` block should never be executed, I think?

If that's correct, then instead of running a value, you should throw an unchecked exception, as it would clearly be a bug in the code if that state was reached!